.github/workflows/issue-assignment-notify.yml

```
@@ -0,0 +1,41 @@
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          script: |
+            const issue = context.payload.issue;
+            const assignee = context.payload.assignee.login;
+
+            await github.rest.issues.createComment({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+              issue_number: issue.number,
+              body: `👋 Hi @${assignee},
+
+Thank you for taking up this issue.
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)